platform/core.network/src/org/netbeans/core/network/proxy/pac/impl/NbPacScriptEvaluator.java

```diff
@@ -213,7 +219,8 @@ public NbPacScriptEvaluator(String pacSourceCocde) throws PacParsingException {
     @Override
     public List<Proxy> findProxyForURL(URI uri) throws PacValidationException {
 
-        List<Proxy> jsResultAnalyzed;
+        AtomicReference<List<Proxy>> resultHolder = new AtomicReference<>(null);
```

Comment:
This can be moved into the `else`-block. No need to hold it in the outer context.